src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -126,14 +121,32 @@ div.modal {
 	font-weight: var(--font-weight-normal);
 
 	div.wrapper.dialog {
-		color: var(--color-white);
+		color: var(--color-secondary);
+		padding-block: var(--padding-3x);
+		padding-inline: var(--padding-0_5x);
+	}
+
+	div.wrapper.dialog div.header {
+		--padding-3x: 0;
```

Comment:
That sounds "dangerous". Any other way to solve the requirement without resetting a core variable?

Reply:
the issue is that in the `Modal` component ([see source code](https://github.com/dfinity/gix-components/blob/1c4ab390f9cab1d0e3ec73a23384e045679eb6b8/src/lib/components/Modal.svelte#L195)), they are hard-coded like that...

either I change it in gix-component and await a release, or I do it like this now. I understand that it is something that we should deploy quickly, so what about this?

- we do it like this PR
- I add a TODO: adjust better when Gix component is updated
- I change GIX component
- we await release and fix this

Reply:
Setting the `padding` does not work?

```
div.wrapper.dialog div.header {
   padding: something
}
```

Reply:
nope, i tried it and it doesn't work... the CSS inside the component will override the one in the main definition file

Reply:
Alright, let's add a TODO

Reply:
cool, done